device/mellanox/x86_64-mlnx_msn2700-r0/Mellanox-SN2700-D48C8/buffers_defaults_objects.j2

```diff
@@ -195,7 +232,32 @@
             "profile" : "[BUFFER_PROFILE|ingress_lossy_profile]"
         }{% if not loop.last %},{% endif %}
 
+{% set pg_loop.last_valid = true %}
+{% else %}
+{% set pg_loop.last_valid = false %}
+{% endif %}
 {% endfor %}
+{% if port_names_extra_pgs|length > 0 %}
+{% if pg_loop.last_valid %},{% endif %}
+{% for port in port_names_extra_pgs.split(',') %}
+{% if dynamic_mode is defined %}
+        "{{ port }}|2-4": {
+            "profile" : "NULL"
+        },
+        "{{ port }}|6": {
+            "profile" : "NULL"
+        },
+{% endif %}
+        "{{ port }}|0": {
+{% if dynamic_mode is defined %}
+            "profile" : "[BUFFER_PROFILE|ingress_lossy_profile]"
+{% else %}
+            "profile" : "[BUFFER_PROFILE|ingress_lossy_profile]"
```

Comment:
profile is the same when `dynamic_mode` is true or not?

Reply:
For lossy, yes. For lossless, it's generated by the template in the dynamic model and by buffer manager in the traditional model

Reply:
Ah, got you. It doesn't make sense to test whether dynamic_mode is enabled or not given that both scenarios result in the same code.